CONTRIBUTING.md

```diff
@@ -824,8 +823,8 @@ rejecting any PR. Although [anyone can review PRs](#reviews), only the
 acknowledgement (or "ack") from an Approver counts towards the approval of a PR.
 
 Approvers are listed in GitHub teams, one for each repository. The project
-uses the [Pull Approve](https://pullapprove.com) service meaning each
-repository contains a top level `.pullapprove.yml` configuration file. This
+uses the [Pull Approve](https://PullApprove.com) service meaning each
```

Comment:
I'm not sure capitalising the URL is correct. I browsed to the capitalised version, and the URL in my browser bar ends up all lowercase....

Reply:
Oh - that was unintentional. But I've removed pullapprove on anther commit anyway if you continue looking ;)